services/column-type-getter.js

```diff
@@ -103,7 +104,7 @@ function ColumnTypeGetter(databaseConnection, schema, allowWarning = true) {
         return 'DOUBLE';
       case 'DATE':
       case 'DATETIME':
-      case 'TIMESTAMP':
+      case (type.match(/^TIMESTAMP.*/i) || {}).input:
       case 'TIMESTAMP WITH TIME ZONE':
       case 'TIMESTAMP WITHOUT TIME ZONE':
```

Comment:
We never reach the last two cases because of the previous line. I kept them to be explicit, but maybe we have to remove them. Let me know what's better!

Reply:
Yes Remove them

Reply:
@VincentMolinie Ok done!